### ISSUE TITLE
Can't touch in SDGB_1.40.

### ISSUE THREAD

**Issue by MoeclubM** [Leapward-Koex/MaiTouchSensorEmulator#6]
The game seems can't find the series port,or is it because there's something wrong with my game settings?

**Comment by Leapward-Koex** [comment]
I haven't tried with any SDGB versions but assuming they're just translated versions of SDEZ I don't see why it shouldn't work.

To check, does the connection state say the app is connected?

And is there any data in the Sent/Received fields?
The received field should start with RSET -> HALT and then finish with STAT when the handshake is complete.
E.g.
![image](https://github.com/Leapward-Koex/MaiTouchSensorEmulator/assets/30615050/b0c9a828-4128-45f8-a83e-ab90e14338ff)
****

**Comment by MoeclubM** [comment]
I checked carefully and found that the serial port created by com2com seemed unavailable on my computer. I used VSPD to manually create the binding between com3 and com23 and then the program became available.

In addition, I have a small suggestion, whether the function of customizing the area can be added, because in actual use, I often encounter the problem of being unable to detect due to clicking too far outside.
